packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6675,7 +6675,7 @@ Update the logo for an application.
 | applicationId   | `id` of an [app](#application-object)      | URL              | String     | TRUE     |
 | file            | File containing logo                       | Body             | File       | TRUE     |
 
-The file must be in PNG, JPG, or GIF format, and less than 1 MB in size. For best results use landscape orientation, a transparent background, and a minimum size of 420px by 120px to prevent upscaling.
+The file must be in PNG, JPG, SVG, or GIF format, and less than 1 MB in size. For best results, use an image with a transparent background and a square dimension of 200px by 200px to prevent scaling.
```

Comment:
I think we can keep `upscaling`? Any reason to remove it?

Reply:
Yes, we can keep it. I'll make the update.